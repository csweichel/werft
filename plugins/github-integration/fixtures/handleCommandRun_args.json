{
    "Event": {
        "action": "edited",
        "changes": {
            "body": {
                "from": "/werft run"
            }
        },
        "issue": {
            "url": "https://api.github.com/repos/csweichel/test-repo/issues/18",
            "repository_url": "https://api.github.com/repos/csweichel/test-repo",
            "labels_url": "https://api.github.com/repos/csweichel/test-repo/issues/18/labels{/name}",
            "comments_url": "https://api.github.com/repos/csweichel/test-repo/issues/18/comments",
            "events_url": "https://api.github.com/repos/csweichel/test-repo/issues/18/events",
            "html_url": "https://github.com/csweichel/test-repo/pull/18",
            "id": 535676932,
            "node_id": "MDExOlB1bGxSZXF1ZXN0MzUxMzQ2ODg5",
            "number": 18,
            "title": "Werft",
            "user": {
                "login": "csweichel",
                "id": 3210701,
                "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/csweichel",
                "html_url": "https://github.com/csweichel",
                "followers_url": "https://api.github.com/users/csweichel/followers",
                "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                "organizations_url": "https://api.github.com/users/csweichel/orgs",
                "repos_url": "https://api.github.com/users/csweichel/repos",
                "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                "received_events_url": "https://api.github.com/users/csweichel/received_events",
                "type": "User",
                "site_admin": false
            },
            "labels": [],
            "state": "open",
            "locked": false,
            "assignee": null,
            "assignees": [],
            "milestone": null,
            "comments": 29,
            "created_at": "2019-12-10T11:46:08Z",
            "updated_at": "2022-02-20T13:18:27Z",
            "closed_at": null,
            "author_association": "OWNER",
            "active_lock_reason": null,
            "draft": false,
            "pull_request": {
                "url": "https://api.github.com/repos/csweichel/test-repo/pulls/18",
                "html_url": "https://github.com/csweichel/test-repo/pull/18",
                "diff_url": "https://github.com/csweichel/test-repo/pull/18.diff",
                "patch_url": "https://github.com/csweichel/test-repo/pull/18.patch",
                "merged_at": null
            },
            "body": "- [ ] /werft foo=bar",
            "reactions": {
                "url": "https://api.github.com/repos/csweichel/test-repo/issues/18/reactions",
                "total_count": 0,
                "+1": 0,
                "-1": 0,
                "laugh": 0,
                "hooray": 0,
                "confused": 0,
                "heart": 0,
                "rocket": 0,
                "eyes": 0
            },
            "timeline_url": "https://api.github.com/repos/csweichel/test-repo/issues/18/timeline",
            "performed_via_github_app": null
        },
        "comment": {
            "url": "https://api.github.com/repos/csweichel/test-repo/issues/comments/1046236299",
            "html_url": "https://github.com/csweichel/test-repo/pull/18#issuecomment-1046236299",
            "issue_url": "https://api.github.com/repos/csweichel/test-repo/issues/18",
            "id": 1046236299,
            "node_id": "IC_kwDOCdZIm84-XEyL",
            "user": {
                "login": "csweichel",
                "id": 3210701,
                "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/csweichel",
                "html_url": "https://github.com/csweichel",
                "followers_url": "https://api.github.com/users/csweichel/followers",
                "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                "organizations_url": "https://api.github.com/users/csweichel/orgs",
                "repos_url": "https://api.github.com/users/csweichel/repos",
                "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                "received_events_url": "https://api.github.com/users/csweichel/received_events",
                "type": "User",
                "site_admin": false
            },
            "created_at": "2022-02-20T13:18:25Z",
            "updated_at": "2022-02-20T13:18:27Z",
            "author_association": "OWNER",
            "body": "/werft run foo=bar arg1\n\n:-1:   ",
            "reactions": {
                "url": "https://api.github.com/repos/csweichel/test-repo/issues/comments/1046236299/reactions",
                "total_count": 0,
                "+1": 0,
                "-1": 0,
                "laugh": 0,
                "hooray": 0,
                "confused": 0,
                "heart": 0,
                "rocket": 0,
                "eyes": 0
            },
            "performed_via_github_app": null
        },
        "repository": {
            "id": 165038235,
            "node_id": "MDEwOlJlcG9zaXRvcnkxNjUwMzgyMzU=",
            "name": "test-repo",
            "full_name": "csweichel/test-repo",
            "private": false,
            "owner": {
                "login": "csweichel",
                "id": 3210701,
                "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/csweichel",
                "html_url": "https://github.com/csweichel",
                "followers_url": "https://api.github.com/users/csweichel/followers",
                "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                "organizations_url": "https://api.github.com/users/csweichel/orgs",
                "repos_url": "https://api.github.com/users/csweichel/repos",
                "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                "received_events_url": "https://api.github.com/users/csweichel/received_events",
                "type": "User",
                "site_admin": false
            },
            "html_url": "https://github.com/csweichel/test-repo",
            "description": "This is just a test repo to play with GitHub apps - nothing to see here",
            "fork": false,
            "url": "https://api.github.com/repos/csweichel/test-repo",
            "forks_url": "https://api.github.com/repos/csweichel/test-repo/forks",
            "keys_url": "https://api.github.com/repos/csweichel/test-repo/keys{/key_id}",
            "collaborators_url": "https://api.github.com/repos/csweichel/test-repo/collaborators{/collaborator}",
            "teams_url": "https://api.github.com/repos/csweichel/test-repo/teams",
            "hooks_url": "https://api.github.com/repos/csweichel/test-repo/hooks",
            "issue_events_url": "https://api.github.com/repos/csweichel/test-repo/issues/events{/number}",
            "events_url": "https://api.github.com/repos/csweichel/test-repo/events",
            "assignees_url": "https://api.github.com/repos/csweichel/test-repo/assignees{/user}",
            "branches_url": "https://api.github.com/repos/csweichel/test-repo/branches{/branch}",
            "tags_url": "https://api.github.com/repos/csweichel/test-repo/tags",
            "blobs_url": "https://api.github.com/repos/csweichel/test-repo/git/blobs{/sha}",
            "git_tags_url": "https://api.github.com/repos/csweichel/test-repo/git/tags{/sha}",
            "git_refs_url": "https://api.github.com/repos/csweichel/test-repo/git/refs{/sha}",
            "trees_url": "https://api.github.com/repos/csweichel/test-repo/git/trees{/sha}",
            "statuses_url": "https://api.github.com/repos/csweichel/test-repo/statuses/{sha}",
            "languages_url": "https://api.github.com/repos/csweichel/test-repo/languages",
            "stargazers_url": "https://api.github.com/repos/csweichel/test-repo/stargazers",
            "contributors_url": "https://api.github.com/repos/csweichel/test-repo/contributors",
            "subscribers_url": "https://api.github.com/repos/csweichel/test-repo/subscribers",
            "subscription_url": "https://api.github.com/repos/csweichel/test-repo/subscription",
            "commits_url": "https://api.github.com/repos/csweichel/test-repo/commits{/sha}",
            "git_commits_url": "https://api.github.com/repos/csweichel/test-repo/git/commits{/sha}",
            "comments_url": "https://api.github.com/repos/csweichel/test-repo/comments{/number}",
            "issue_comment_url": "https://api.github.com/repos/csweichel/test-repo/issues/comments{/number}",
            "contents_url": "https://api.github.com/repos/csweichel/test-repo/contents/{+path}",
            "compare_url": "https://api.github.com/repos/csweichel/test-repo/compare/{base}...{head}",
            "merges_url": "https://api.github.com/repos/csweichel/test-repo/merges",
            "archive_url": "https://api.github.com/repos/csweichel/test-repo/{archive_format}{/ref}",
            "downloads_url": "https://api.github.com/repos/csweichel/test-repo/downloads",
            "issues_url": "https://api.github.com/repos/csweichel/test-repo/issues{/number}",
            "pulls_url": "https://api.github.com/repos/csweichel/test-repo/pulls{/number}",
            "milestones_url": "https://api.github.com/repos/csweichel/test-repo/milestones{/number}",
            "notifications_url": "https://api.github.com/repos/csweichel/test-repo/notifications{?since,all,participating}",
            "labels_url": "https://api.github.com/repos/csweichel/test-repo/labels{/name}",
            "releases_url": "https://api.github.com/repos/csweichel/test-repo/releases{/id}",
            "deployments_url": "https://api.github.com/repos/csweichel/test-repo/deployments",
            "created_at": "2019-01-10T10:17:50Z",
            "updated_at": "2021-07-06T13:05:32Z",
            "pushed_at": "2022-02-11T17:16:15Z",
            "git_url": "git://github.com/csweichel/test-repo.git",
            "ssh_url": "git@github.com:csweichel/test-repo.git",
            "clone_url": "https://github.com/csweichel/test-repo.git",
            "svn_url": "https://github.com/csweichel/test-repo",
            "homepage": null,
            "size": 115,
            "stargazers_count": 0,
            "watchers_count": 0,
            "language": "Dockerfile",
            "has_issues": true,
            "has_projects": true,
            "has_downloads": true,
            "has_wiki": true,
            "has_pages": true,
            "forks_count": 1,
            "mirror_url": null,
            "archived": false,
            "disabled": false,
            "open_issues_count": 9,
            "license": null,
            "allow_forking": true,
            "is_template": false,
            "topics": [],
            "visibility": "public",
            "forks": 1,
            "open_issues": 9,
            "watchers": 0,
            "default_branch": "master"
        },
        "sender": {
            "login": "werft-ci[bot]",
            "id": 58423842,
            "node_id": "MDM6Qm90NTg0MjM4NDI=",
            "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/werft-ci%5Bbot%5D",
            "html_url": "https://github.com/apps/werft-ci",
            "followers_url": "https://api.github.com/users/werft-ci%5Bbot%5D/followers",
            "following_url": "https://api.github.com/users/werft-ci%5Bbot%5D/following{/other_user}",
            "gists_url": "https://api.github.com/users/werft-ci%5Bbot%5D/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/werft-ci%5Bbot%5D/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/werft-ci%5Bbot%5D/subscriptions",
            "organizations_url": "https://api.github.com/users/werft-ci%5Bbot%5D/orgs",
            "repos_url": "https://api.github.com/users/werft-ci%5Bbot%5D/repos",
            "events_url": "https://api.github.com/users/werft-ci%5Bbot%5D/events{/privacy}",
            "received_events_url": "https://api.github.com/users/werft-ci%5Bbot%5D/received_events",
            "type": "Bot",
            "site_admin": false
        },
        "installation": {
            "id": 5647067,
            "node_id": "MDIzOkludGVncmF0aW9uSW5zdGFsbGF0aW9uNTY0NzA2Nw=="
        }
    },
    "PR": {
        "id": 351346889,
        "number": 18,
        "head": {
            "label": "csweichel:werft",
            "ref": "werft",
            "sha": "37cb8b8f7fc3499bdf65d662793dd2d968ae3bac",
            "repo": {
                "id": 165038235,
                "node_id": "MDEwOlJlcG9zaXRvcnkxNjUwMzgyMzU=",
                "owner": {
                    "login": "csweichel",
                    "id": 3210701,
                    "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                    "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                    "html_url": "https://github.com/csweichel",
                    "gravatar_id": "",
                    "type": "User",
                    "site_admin": false,
                    "url": "https://api.github.com/users/csweichel",
                    "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                    "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                    "followers_url": "https://api.github.com/users/csweichel/followers",
                    "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                    "organizations_url": "https://api.github.com/users/csweichel/orgs",
                    "received_events_url": "https://api.github.com/users/csweichel/received_events",
                    "repos_url": "https://api.github.com/users/csweichel/repos",
                    "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                    "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions"
                },
                "name": "test-repo",
                "full_name": "csweichel/test-repo",
                "description": "This is just a test repo to play with GitHub apps - nothing to see here",
                "default_branch": "master",
                "created_at": "2019-01-10T10:17:50Z",
                "pushed_at": "2022-02-11T17:16:15Z",
                "updated_at": "2021-07-06T13:05:32Z",
                "html_url": "https://github.com/csweichel/test-repo",
                "clone_url": "https://github.com/csweichel/test-repo.git",
                "git_url": "git://github.com/csweichel/test-repo.git",
                "ssh_url": "git@github.com:csweichel/test-repo.git",
                "svn_url": "https://github.com/csweichel/test-repo",
                "language": "Dockerfile",
                "fork": false,
                "forks_count": 1,
                "open_issues_count": 9,
                "stargazers_count": 0,
                "watchers_count": 0,
                "size": 115,
                "archived": false,
                "disabled": false,
                "private": false,
                "has_issues": true,
                "has_wiki": true,
                "has_pages": true,
                "has_projects": true,
                "has_downloads": true,
                "is_template": false,
                "url": "https://api.github.com/repos/csweichel/test-repo",
                "archive_url": "https://api.github.com/repos/csweichel/test-repo/{archive_format}{/ref}",
                "assignees_url": "https://api.github.com/repos/csweichel/test-repo/assignees{/user}",
                "blobs_url": "https://api.github.com/repos/csweichel/test-repo/git/blobs{/sha}",
                "branches_url": "https://api.github.com/repos/csweichel/test-repo/branches{/branch}",
                "collaborators_url": "https://api.github.com/repos/csweichel/test-repo/collaborators{/collaborator}",
                "comments_url": "https://api.github.com/repos/csweichel/test-repo/comments{/number}",
                "commits_url": "https://api.github.com/repos/csweichel/test-repo/commits{/sha}",
                "compare_url": "https://api.github.com/repos/csweichel/test-repo/compare/{base}...{head}",
                "contents_url": "https://api.github.com/repos/csweichel/test-repo/contents/{+path}",
                "contributors_url": "https://api.github.com/repos/csweichel/test-repo/contributors",
                "deployments_url": "https://api.github.com/repos/csweichel/test-repo/deployments",
                "downloads_url": "https://api.github.com/repos/csweichel/test-repo/downloads",
                "events_url": "https://api.github.com/repos/csweichel/test-repo/events",
                "forks_url": "https://api.github.com/repos/csweichel/test-repo/forks",
                "git_commits_url": "https://api.github.com/repos/csweichel/test-repo/git/commits{/sha}",
                "git_refs_url": "https://api.github.com/repos/csweichel/test-repo/git/refs{/sha}",
                "git_tags_url": "https://api.github.com/repos/csweichel/test-repo/git/tags{/sha}",
                "hooks_url": "https://api.github.com/repos/csweichel/test-repo/hooks",
                "issue_comment_url": "https://api.github.com/repos/csweichel/test-repo/issues/comments{/number}",
                "issue_events_url": "https://api.github.com/repos/csweichel/test-repo/issues/events{/number}",
                "issues_url": "https://api.github.com/repos/csweichel/test-repo/issues{/number}",
                "keys_url": "https://api.github.com/repos/csweichel/test-repo/keys{/key_id}",
                "labels_url": "https://api.github.com/repos/csweichel/test-repo/labels{/name}",
                "languages_url": "https://api.github.com/repos/csweichel/test-repo/languages",
                "merges_url": "https://api.github.com/repos/csweichel/test-repo/merges",
                "milestones_url": "https://api.github.com/repos/csweichel/test-repo/milestones{/number}",
                "notifications_url": "https://api.github.com/repos/csweichel/test-repo/notifications{?since,all,participating}",
                "pulls_url": "https://api.github.com/repos/csweichel/test-repo/pulls{/number}",
                "releases_url": "https://api.github.com/repos/csweichel/test-repo/releases{/id}",
                "stargazers_url": "https://api.github.com/repos/csweichel/test-repo/stargazers",
                "statuses_url": "https://api.github.com/repos/csweichel/test-repo/statuses/{sha}",
                "subscribers_url": "https://api.github.com/repos/csweichel/test-repo/subscribers",
                "subscription_url": "https://api.github.com/repos/csweichel/test-repo/subscription",
                "tags_url": "https://api.github.com/repos/csweichel/test-repo/tags",
                "trees_url": "https://api.github.com/repos/csweichel/test-repo/git/trees{/sha}",
                "teams_url": "https://api.github.com/repos/csweichel/test-repo/teams",
                "visibility": "public"
            },
            "user": {
                "login": "csweichel",
                "id": 3210701,
                "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                "html_url": "https://github.com/csweichel",
                "gravatar_id": "",
                "type": "User",
                "site_admin": false,
                "url": "https://api.github.com/users/csweichel",
                "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                "followers_url": "https://api.github.com/users/csweichel/followers",
                "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                "organizations_url": "https://api.github.com/users/csweichel/orgs",
                "received_events_url": "https://api.github.com/users/csweichel/received_events",
                "repos_url": "https://api.github.com/users/csweichel/repos",
                "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions"
            }
        }
    }
}