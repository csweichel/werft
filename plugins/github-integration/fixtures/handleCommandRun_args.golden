{
  "req": {
    "metadata": {
      "owner": "csweichel",
      "repository": {
        "host": "github.com",
        "owner": "csweichel",
        "repo": "test-repo",
        "ref": "refs/heads/werft",
        "revision": "37cb8b8f7fc3499bdf65d662793dd2d968ae3bac",
        "defaultBranch": "master"
      },
      "trigger": "TRIGGER_MANUAL",
      "annotations": [
        {
          "key": "updateGitHubStatus",
          "value": "csweichel/test-repo"
        },
        {
          "key": "foo",
          "value": "bar"
        },
        {
          "key": "arg1"
        }
      ]
    },
    "spec": {
      "jobPath": ""
    }
  },
  "msg": "started the job as [foo](/job/foo)"
}